loss_probability:
  rate: 20
  multiagent_topics_pre: /uavs/agent_
  visibility_matrix_topic_post: /visibility_matrix
  probability_topic: /uavs/leader/target_loss_probability
  target_pose_topic: /target/odometry

  agents_no: 5
  prediction_interval: 0.5
  prior:
    state: [1, 0, 0, 0]
    error: [1, 0, 0, 0,
            0, 1, 0, 0,
            0, 0, 1, 0,
            0, 0, 0, 1]
  kalman_model:
    sample_time: 0.05
    beta: 5
    sigma: 0.01